components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -409,14 +413,38 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 		}
 	}
 
-	__reactToChanges() {
-		const contentRect = this.__content.getBoundingClientRect();
-		if (contentRect.height < 1) return;
-		const eventDetails = {
-			bubbles: true,
-			composed: true,
-			detail: contentRect
-		};
-		this.dispatchEvent(new CustomEvent('d2l-hierarchical-view-resize', eventDetails));
+	__reactToMutationChanges() {
+		this.__isChildView();
+
+		if (this.childView) {
+			this.__removeEventListeners();
```

Comment:
Previously we would have only added the listeners for the root view, so it looks like we need to remove them here since we may have wired them up when we should not have (due to delayed detection of child-view/parent-view)... is that correct?

Reply:
Yup that's correct